index.html

```
@@ -2,60 +2,134 @@
+        <div class="stat-item stat-item--card">
+          <div class="stat-icon">
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2">
+              <path d="M20.59 13.41l-7.17 7.17a2 2 0 0 1-2.83 0L2 12V2h10l8.59 8.59a2 2 0 0 1 0 2.82z"/>
+              <line x1="7" y1="7" x2="7.01" y2="7"/>
+            </svg>
+          </div>
+          <span class="stat-number stat-number--animated" data-target="{{ site.tags.size }}">{{ site.tags.size }}</span>
+          <span class="stat-label" data-i18n="hero.tags">Тегов</span>
+          <div class="stat-indicator stat-indicator--active"></div>
         </div>
       </dl>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </dl> ]

(tag-pair)